bitnami/keycloak/README.md

```diff
@@ -560,6 +576,12 @@ Adding the TLS parameter (where available) will cause the chart to generate HTTP
 
 [Learn more about Ingress controllers](https://kubernetes.io/docs/concepts/services-networking/ingress-controllers/).
 
+### Configure admin Ingress
+
+In addition to the Ingress resource described above, this chart also provides the ability to define an Ingress for the admin area of Keycloak, for example the `master` realm.  
+
+For this scenario, set the environment variable `KC_HOSTNAME_ADMIN` to the appropriate value (e.g. `keycloak-admin.example.com`) and the `frontendUrl` property of the realm accordingly (e.g. `https://keycloak-admin.example.com`).
```

Comment:
If the `KC_HOSTNAME_ADMIN` is automated, I would adapt this line.

Reply:
Also, maybe it can be nice an example of how to adapt the `frontendUrl` property in _values.yaml_

Reply:
I've just committed a change to set `KC_HOSTNAME_ADMIN` automatically.

But I'm wondering what you meant regarding `frontendUrl` and `values.yaml`.
For the record, the way I managed to automate the configuration of the `frontendUrl` property of the `master` realm, was with the Keycloak CLI functionality of the chart.

Using a simple CLI configuration file as:
```
{
  "realm" : "master",
  "attributes": {
    "frontendUrl": "$(env:MASTER_REALM_FRONTEND_URL)"
  }
}
```

And defining the environment variable `MASTER_REALM_FRONTEND_URL=https://keycloak-admin.example.com` so that the value can be substituted.

Or is there a simpler mechanism that I missed?

Reply:
My bad, I might have not be clear enough. I meant that it would be nice to have an example in _README.md_ about how to modify `values.yaml` so that the `frontendUrl` points to the right place. Roughly something like:

```
Set `keycloakConfigCli.configuration` to the following:

configuration:
{
  "realm" : "master",
  "attributes": {
    "frontendUrl": "YOUR_URI"
  }
}
```

At the end, I only want that people using the feature have clear instructions on how to set everything up 😁 . I hope it is clear now.